docs/install.md

```diff
@@ -122,6 +122,45 @@ Select the following tabs for the installation description of your target.
 
     is needed. A recommended size is at least a free space of 10 GiB.
 
+    ??? important "Installation of VirtualBox with VirtualBox extension pack ("Guest additions")"
```

Comment:
Little not here to not forget checking if the doubled double quotes cause issues here. Probably they need to be escaped or single quotes used, then the text line is terminated at the second double quote.

Reply:
At least the preview via `mkdocs serve` worked well.